ratelimiter/_async.py

```diff
@@ -32,4 +32,5 @@ async def __aenter__(self):
                     await asyncio.sleep(sleeptime)
             return self
 
-    __aexit__ = asyncio.coroutine(RateLimiter.__exit__)
+    # backward behavior for python 3.5 - 3.10; no need for decorator if deprecated in python > 3.11
+    __aexit__ = asyncio.coroutine(RateLimiter.__exit__) if hasattr(asyncio, 'coroutine') else RateLimiter.__exit__
```

Comment:
Hey @brainysmurf - have you tested whether returning `RateLimiter.__exit__` behaves as expected on Python 3.11?  (it's [declared using `def`](https://github.com/RazerM/ratelimiter/blob/da78a45867c3a204083c6ea8ee74f6b3e78ef524/ratelimiter/_sync.py#L63), not `async def`, which is what has me wondering)

Reply:
Doesn't work as expected. Tried implementing then changes and when called I get 

`name 'RateLimiter' is not defined`

And the error points to "RateLimiter.__exit__"